cmd/eval.go

```diff
@@ -64,7 +73,7 @@ Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
 				return fmt.Errorf("failed to access replacements flag: %w", err)
 			}
 
-			file, err := os.ReadFile(args[0])
+			file, err := os.Open(args[0])
```

Comment:
minor and very optional: sometimes it's useful to assign indices as constants. E.g. `const templatePathIndex = 0` then when you do file, err := os.Open(args[templatePathIndex]) you know what it is retrieving from the args. You could also just put it in a var, e.g. `templatePlat := args[0]` or some such. Just a very minor suggestion.

Reply:
suggestion: maybe also the `file` variable name is too generic? `comboTemplateFile` or something like that could be better?

Reply:
Nice yeah, these are clean suggestions. Let me throw up an implementation.